packages/common-common/src/redisCache.ts

```diff
@@ -1,5 +1,6 @@
 import { ConnectionTimeoutError, createClient, ReconnectStrategyError, SocketClosedUnexpectedlyError } from "redis";
 import { factory, formatFilename } from './logging';
+import { REDIS_URL, VULTR_IP } from 'commonwealth/server/config';
```

Comment:
Another back-dependency we shouldn't be doing.

Reply:
for rich/tim

Reply:
these were unused imports, so just removed them